src/Hangman/HangmanGame.java

```diff
@@ -0,0 +1,208 @@
+package Hangman;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Scanner;
+
+public class HangmanGame {
+
+    private String[] board;
+
+    private static String word;
+
+    private static int lengthOfWord;
+
+    private static String[] lettersToGuess;
+
+    private static int incorrectGuesses = 0;
+
+    private static int correctGuesses = 0;
+    private static HashSet<String> lettersGuessed;
+
+
+
+    public HangmanGame(){
+        board = new String[]{"  ┌──┐",
+                             "     │",
+                             "     │",
+                             "     │",
+                             "   ──┴──"};
+        lettersGuessed = new HashSet<String>();
+    }
+
+    public void printBeginningBoard(){
```

Comment:
This code design works, but i think a better implementation that'll make it easier to debug, maintain, and easier to read is to have a single private variable that represents the hangman board. That way we can call/print/add/delete that single variable. Instead of having for instance printBeginningBoard and printEndBoard essentially doing the same thing.

Kind of like a one source of truth for the gameboard instead of having different places in the code where the board could be created or manipulated.